.circleci/deploy-config.yml

```diff
@@ -285,23 +301,23 @@ workflows:
     - run_new_imports:
         name: "Development: Run New Imports"
         requires: [build_and_test]
-        context: [deployment-development-wdiv]
+        context: [deployment-development-wdiv, slack-secrets]
```

Comment:
I think you also need to add `context: [slack-secrets]` to the `build_and_test` job above (line 297), and below (line 328).